language/python/build-images.md

```diff
@@ -4,8 +4,238 @@ keywords: python, build, images, dockerfile
 description: Learn how to build your first Docker image by writing a Dockerfile
 ---
 
-> ### Coming soon
+{% include_relative nav.html selected="1" %}
+
+## Prerequisites
+
+Work through the orientation and setup in Get started [Part 1](/get-started/) to understand Docker concepts.
+
+## Overview
+
+Now that we have a good overview of containers and the Docker platform, let’s take a look at building our first image. An image includes everything needed to run an application - the code or binary, runtime, dependencies, and any other file system objects required.
+
+To complete this tutorial, you need the following:
+
+- Python version 3.8 or later. [Download Python](https://www.python.org/downloads/){: target="_blank" rel="noopener" class="_"}
+- Docker running locally. Follow the instructions to [download and install Docker](../../desktop/index.md)
+- An IDE or a text editor to edit files. We recommend using [Visual Studio Code](https://code.visualstudio.com/Download){: target="_blank" rel="noopener" class="_"}.
+
+## Sample application
+
+Let’s create a simple Python application using the Flask framework that we’ll use as our example. Create a directory in your local machine named `python-docker` and follow the steps below to create a simple web server.
+
+```shell
+$ cd /path/to/python-docker
+$ pip install Flask
+$ pip freeze > requirements.txt
+$ touch app.py
+```
+
+Now, let’s add some code to handle simple web requests. Open this working directory in your favorite IDE and enter the following code into the `app.py` file.
+
+```shell
+from flask import Flask
+app = Flask(__name__)
+
+@app.route('/')
+def hello_world():
+    return 'Hello, Docker!'
+```
+
+## Test the application
+
+Let’s start our application and make sure it’s running properly. Open your terminal and navigate to the working directory you created.
+
+```shell
+$ python -m flask run
```

Comment:
Same suggestion to use `python3` instead of `python`.
On my Mac I have eg. two version of Pythons installed, don't know when and why I have them, more because it was needed to run some tools. I'm not a Python developer, so I'm always confused 😅